contrib/NeonVM/neonvm.c

```diff
@@ -732,6 +804,86 @@ void exec_CALLP(struct tagTExecutor *self)
         push(self->stack, a);
         cell_freeCell(left);
         cell_freeCell(right);
+    } else if (strcmp(func, "array__size") == 0) {

```

Comment:
I think it's definitely time to break each of these out into their own functions!